var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -85,6 +87,7 @@ class Openmpi(Package):
     variant('verbs', default=_verbs_dir() is not None,
             description='Build support for OpenFabrics verbs.')
     variant('mxm', default=False, description='Build Mellanox Messaging support')
+    variant('cxx', default=False, description='Enable deprecated C++ bindings.')
```

Comment:
any reason for this variant? why not have it always `on`? If you really need an option to turn it `off`, i would keep it `True` by default so that the default installation is the same as before this PR.